models/games.model.js

```diff
@@ -44,4 +45,21 @@ exports.checkReviewIdExist = (review_id) => {
             }
             return [];
         })
-};
\ No newline at end of file
+};
+
+exports.insertCommentsByReviewId = (comment, id) => {
+    const { username, body } = comment;
+    const { review_id } = id
+    return db
+        .query(`INSERT INTO comments 
+        (author, body, review_id) 
+        VALUES 
+        ($1, $2, $3) 
+        RETURNING *;`,
+            [username, body, review_id]
+        )
+        .then(({ rows }) => {
+            return (rows[0]);
+        });
```

Comment:
Looks good so far, will just need to add some more code to get the missing two tests to pass